charts/operator-wandb/templates/_mysql.tpl

```diff
@@ -1,9 +1,36 @@
 {{/*
-Return name of secret where information is stored
+Return the name of the secret where information is stored, considering if the custom secret is defined
 */}}
 {{- define "wandb.mysql.passwordSecret" -}}
-{{- print .Release.Name "-mysql" -}}
+{{- if .Values.global.mysql.passwordSecret.name }}
+  {{- .Values.global.mysql.passwordSecret.name -}}
+{{- else }}
+  {{- print .Release.Name "-mysql" -}}
 {{- end -}}
+{{- end -}}
+
+{{/*
+Return the key of the secret where the password is stored, considering if the custom key is defined
+*/}}
+{{- define "wandb.mysql.passwordSecret.key" -}}
+{{- if .Values.global.mysql.passwordSecret.userKey }}
+  {{- .Values.global.mysql.passwordSecret.userKey -}}
```

Comment:
I dont think this is neccessary since `.Values.global.mysql.passwordSecret.userKey` has a default value it will never be falsey, this define is probably redundant